frontends/web/src/locales/en/app.json

```diff
@@ -2,6 +2,8 @@
   "account": {
     "account": "Account",
     "accounts": "Accounts",
+    "backupReminder": "Your wallet <strong>{{name}}</strong> just passed {{fiat}} 1000!\n\nWe recommend creating a paper backup for extra protection. It's quick and simple.",
```

Comment:
I'd remove "just", as it will be shown to any user who has had more than 1000 even for a long time.

Also, please make the variable contain "USD 1000", including the amount - we don't need the amount to be translated, and when we want to change it re-translate it.

Also, would be good to apply `coin.FormatAsCurrency()` to the actual threshold in the backend and `formatLocalizedAmount()` in the frontend, as we do with other fiat amounts.

Reply:
I'm not entirely sure I follow what's the proper way to handle this. I made some changes but I'm not sure, especially on why we need to format in the backend as we don't really use the string there. However, please take a look and let me know if what I did now works for you, thanks!